src/nyc_trees/apps/mail/views.py

```diff
@@ -72,10 +82,16 @@ def notify_rsvp(request, user, event):
         'event_slug': event.slug
     })
     event_url = request.build_absolute_uri(relative_event_url)
+    if event.map_pdf_filename:
```

Comment:
I think this will fail if an email is sent while the map is being (re)built.

The filename gets set at the start of the map generation process, so there is a window of time when the file path points to a non-existent file.
The answer is to also check if the file exists, like so: `default_storage.exists(event.map_pdf_filename)`